### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by XandraMcC**
- This PR contains the Dashboard GUI 
- Built with ReactJS (with NextJS)
- Contains 3 pages (in this initial PR): Jobs Dashboard; Nodes Dashboard; Settings
- Ability to view Jobs and Nodes using the Bacalhau api
- Ability to customise Nodes Dashboard in the settings page

Next:
- Need the ability to spin up Dashboard & configure port in CLI

### CODE REVIEW

File: dashboard/src/assets/fonts/Open_Sans/README.txt [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Open Sans Variable Font
+=======================
+
+This download contains Open Sans as both variable fonts and static fonts.
```
Comment (simonwo) [inline dashboard/src/assets/fonts/Open_Sans/README.txt:4]: Do we really have to include all of these fonts in the source code? Can't we use some NPM package for this? Does [`@fontsource/open-sans`](https://www.npmjs.com/package/@fontsource/open-sans) work?

File: dashboard/src/assets/images/vercel.svg
Comment (simonwo) [inline dashboard/src/assets/images/vercel.svg:1]: Does this need to be here?

**Review by simonwo — COMMENTED** [review]
Can we have some instructions about how to actually spin up the dashboard, pre-embedding it in the app?

File: dashboard/src/assets/fonts/Open_Sans/README.txt (hunk 1 above)
Comment (XandraMcC) [reply dashboard/src/assets/fonts/Open_Sans/README.txt]: replaced

File: dashboard/src/assets/images/vercel.svg
Comment (XandraMcC) [reply dashboard/src/assets/images/vercel.svg:1]: removed

**Comment by simonwo** [comment]
Replaced by #3044.

**Pull request closed without merging**